packages/nx/src/command-line/graph/graph.ts

```diff
@@ -749,6 +785,10 @@ async function createExpandedTaskInputResponse(
   return response;
 }
 
+function createSourceMapsResponse() {
+  return readSourceMaps();
```

Comment:
This reads it from the file? Can we make a new function which will build a graph and return the source map?